articles/sentinel/process-events-normalization-schema.md

```diff
@@ -178,16 +179,16 @@ The process event schema references the following entities, which are central to
 | **ActingProcessCreationTime**      | Optional     | DateTime   |       The date and time when the acting process was started. |
 | **ActingProcessTokenElevation**    | Optional     | String     | A token indicating the presence or absence of User Access Control (UAC) privilege elevation applied to the acting process.   <br><br>Example:  `None`|
 | **ActingProcessFileSize**          | Optional     | Long       |      The size of the file that ran the acting process.   |
-| **ParentProcessName**              | Optional     | string     |  The file name of the parent process image file. This is considered the process name.    <br><br>Example: `C:\Windows\explorer.exe` |
-| **ParentProcessCompany**       | Optional     | String     |The name of the company that created the parent process image file.            <br><br>    Example:  `Microsoft`   |
+| **ParentProcessName**              | Optional     | string     | The name of the parent process, commonly derived from the image file or executable file used to define the initial code and data mapped into the process' virtual address space.<br><br>Example: `C:\Windows\explorer.exe` |
```

Comment:
`The name of the parent process. This name is commonly derived from the image or executable file that's used to define the initial code and data that's mapped into the process' virtual address space.`